staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/request/admissionreview_test.go

```diff
@@ -0,0 +1,618 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package request
+
+import (
+	"reflect"
+	"strings"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+
+	admissionv1 "k8s.io/api/admission/v1"
+	admissionv1beta1 "k8s.io/api/admission/v1beta1"
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
```

Comment:
could you remind me why we are still using v1beta1 internally? Do we have plan to switch the configuration manager and the webhook packages to use v1?

Reply:
Yes, we will. Just wanted to keep this one focused on the admissionreview API

Reply:
opened https://github.com/kubernetes/kubernetes/pull/80883 to do so, though I think we'll hold that for 1.17